Sources/PathKit.swift

```diff
@@ -152,8 +152,18 @@ extension Path {
   ///
   public func abbreviate() -> Path {
 #if os(Linux)
-    // TODO: actually de-normalize the path
-    return self
+    let env = ProcessInfo.processInfo.environment
+    guard let home = env["HOME"], !home.isEmpty else { return self }
+    guard let homeRange = self.path.range(of: home, options: [.anchored, .caseInsensitive]) else { return self }
```

Comment:
I think `caseInsensitive` can be a problem for case sensitive file systems (usually default on linux). Although, I'm not sure how we can check.

We could add a method to normalise a paths case (similar to https://docs.python.org/3/library/os.path.html#os.path.normcase) which then we can normalise both `HOME` and the path before we check (case sensitively).